tools/storybook-addon/src/main/index.ts

```diff
@@ -72,15 +56,36 @@ export const webpackFinal: StorybookConfig['webpackFinal'] = config => {
   ) as RuleSetRule | undefined;
 
   if (fileLoaderRule) {
-    fileLoaderRule.exclude = '/\.svg$/';
+    fileLoaderRule.exclude = /\.svg$/;
   }
 
   config.module.rules.push({
-    test: '/\.svg$/',
+    test: /\.svg$/,
     enforce: 'pre',
     loader: require.resolve('@svgr/webpack'),
   });
 
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
```

Comment:
For whatever reason, the default storybook webpack config (with an explicit `babel` export) doesn't apply to `.ts` files